### DISCUSSION TITLE
Commonware Christmas poster design, please provide feedback if you have any suggestions.

### DISCUSSION THREAD

**Discussion by JackLamCHN** [commonwarexyz/monorepo#248] [Ideas]
Hello everyone, I asked the designers on my team to create a Christmas-themed poster for Commonware:

1、The designer expanded on the logo elements from the top left corner of the project’s official website and incorporated Christmas-themed materials. I summarized the main features of the project into a slogan.

2、Everyone can take a look, and if you have any suggestions for modifications, please let me know. I will ask the designer to make adjustments. Once there are no issues with the modifications or by December 17th if there are no further suggestions, I will share the poster image and source files here.

3、Regarding copyright, as long as everyone uses it for positive promotion of the Commonware project, the copyright is completely free and open, and you are welcome to create derivative works.

4、If Commonware needs any poster designs or video tutorials in the future, the designers in my team can create them. Please let me know if there is a need. Thank you.

**5、Revision History:**
**a.** 2024-12-14 Based on @patrick-ogrady's suggestion, the Discord link has been removed. The designer recommended moving the project link to the central area for better page balance and designing the previously empty section. 

**b.** 2024-12-15, based on @patrick-ogrady's suggestion, the slogan in the image has been changed, and two new versions have been created. The latest version is shown in the image below.

**Commonware Christmas poster source file**
According to the agreed time, I will share the source file (AI format) of this Christmas poster on Google Drive. If you need it, please feel free to download and use it. If there are any additional modifications needed, please let me know. I want to emphasize again that anyone planning to use the poster materials for positive and constructive promotion of Commonware can use them for free for derivative creations.
https://drive.google.com/drive/folders/1GC7S61YrTFMPETZxWCxvHBq6qxGvoxkY?usp=sharing

**Rethinking How the Best Builders Ship Onchain**
![Rethinking How the Best Builders Ship Onchain](https://github.com/user-attachments/assets/e27c1ffc-e080-4dda-a6a0-9742ab9e60c7)

**More powerful onchain apps**
![More powerful onchain apps](https://github.com/user-attachments/assets/afab1a26-d14a-4fb2-b98c-fa62f0b41b38)

**Comment by patrick-ogrady** [comment]
That's a clever idea to replace the edge elements with different icons as needed for the occasion. Have been pondering how I might eventually theme the logo and this might be the path lol.

(Quick nit: can you remove the discord invite for now to avoid confusion?)

**Reply by JackLamCHN** [reply]
Adjustments have been made based on your suggestions. Please take a look at the latest version of the poster.👆

**Reply by patrick-ogrady** [reply]
I think the "slogan" you want to use is definitely up to you (as the creator). That being said, I wouldn't say the current phrase really emphasizes what we are thinking about/focusing on.

I'd recommend something like:
* "More powerful onchain apps"
* "Rethinking How the Best Builders Ship Onchain"
* etc.

Looks cool in any case!

**Reply by JackLamCHN** [reply]
Based on your suggestions, I have made changes to two versions with entirely new phrases. Please check the updates above. 👆

**Comment by kristie-dude** [comment]
Looks nice. I would try to reduce the number of colors used (7), IMO it would look cleaner. For example, all places with that bright beige color could be replaced with white (candy usually is red and white). maybe pink can be replaced with red or white depending on the segment composition. The green base of the snowball is different green than other green elements. Hope it helps! Regards!

**Reply by JackLamCHN** [reply]
Please check.

![Commonware user 1](https://github.com/user-attachments/assets/db22bd63-8d14-4fef-ad45-ba9dcf420a3f)

**Comment by JackLamCHN** [comment]
💡 The poster design work has been completed.
